src/main/resources/treeformats.yml

```diff
@@ -79,5 +79,5 @@ formats:
       regex: "^(.+?)[=\\s]"
       group: 1
     version:
-      regex: "(==|installed:\\s)([^\\]]+)\\]?$"
+      regex: "(==|installed:\\s)([^\\]|,]+)\\]?$"
```

Comment:
This regex will match at the _first_ version number, instead of the "installed" version number.

Although this will work and provide the desired output, it does not make the regex any clearer.

My suggestion is to add a `\\S` (="not a space") before the `==` instead, so that part will never match on the "required: ==0.3.3" string.

Reply:
Changed this. :)